src/Api/Serializer/AbstractSerializer.php

```diff
@@ -280,4 +313,27 @@ public static function setContainer(Container $container)
     {
         static::$container = $container;
     }
+
+    /**
+     * @param string $serializerClass
+     * @param callable $attributeHandler
+     */
+    public static function addAttributeHandler(string $serializerClass, callable $attributeHandler)
+    {
+        if (! isset(static::$attributeHandlers[$serializerClass])) {
+            static::$attributeHandlers[$serializerClass] = [];
+        }
+
+        static::$attributeHandlers[$serializerClass][] = $attributeHandler;
```

Comment:
Actually I don't think it's necessary to check ? php doesn't seem to throw an error.

Reply:
Really? That's an interesting quirk. I'd say let's keep it in to be explicit though